pkg/rtc/utils.go

```diff
@@ -198,3 +201,16 @@ func LoggerWithCodecMime(l logger.Logger, mime string) logger.Logger {
 	}
 	return l
 }
+
+func MaybeTruncateIP(addr string) string {
+	ipAddr := net.ParseIP(addr)
+	if ipAddr == nil {
+		return ""
+	}
+
+	if ipAddr.IsPrivate() || len(addr) <= cMinIPTruncateLen {
+		return addr
+	}
+
+	return addr[:len(addr)-3] + "..."
```

Comment:
Isn't possible to still get `out of range`?
Recently we got:
```
cloud-media panic: runtime error: slice bounds out of range [:-1]                                                                                                                                                                                                                                            
cloud-media goroutine 1123224 [running]:                                                                                                                                                                                                                                                                     
cloud-media github.com/livekit/livekit-server/pkg/rtc.wrappedICECandidatePairLogger.MarshalLogObject({0xc0741cc000?}, {0x2af01e0, 0xc00948e480})                                                                                                                                                             
cloud-media     /go/pkg/mod/github.com/livekit/livekit-server@v1.8.1-0.20241025132723-49b75e94a6dd/pkg/rtc/transport.go:156 +0x417                                                                                                                                                                           
```
what looks like `len(addr)` was `2`